packages/functional-tests/pages/login.ts

```diff
@@ -23,7 +23,10 @@ export const selectors = {
   SHOW_PASSWORD: '#password ~ [for="show-password"]',
   RESET_PASSWORD_EXPIRED_HEADER: '#fxa-reset-link-expired-header',
   RESET_PASSWORD_HEADER: '#fxa-reset-password-header',
+  SIGN_UP_CODE_HEADER: '#fxa-confirm-signup-code-header',
```

Comment:
Unrelated to this line, but we should put a copyright header in this file.

Reply:
fixed.